timemachine/cpp/src/gpu_utils.cuh

```diff
@@ -24,7 +24,9 @@ curandStatus_t templateCurandUniform(curandGenerator_t generator, float *outputP
 curandStatus_t templateCurandUniform(curandGenerator_t generator, double *outputPtr, size_t n);
 
 #define gpuErrchk(ans)                                                                                                 \
-    { gpuAssert((ans), __FILE__, __LINE__); }
+    {                                                                                                                  \
```

Comment:
this looks kind of weird - esp with the end of line line-breaks (`\`) - maybe `#defines` aren't well handled?

Reply:
Will look into it, there might be an option controlling the behavior here.

Reply:
not a big deal tbh

Reply:
I'm not seeing an obvious way to avoid this (though might be missing something, `clang-format` has a [bajillion options](https://clang.llvm.org/docs/ClangFormatStyleOptions.html)). Will leave as-is if not a big deal.